### ISSUE TITLE
Error at macs2 stage on resumed caper run

### ISSUE THREAD

**Issue by jwtobias** [ENCODE-DCC/atac-seq-pipeline#140]
On a resumed caper run the pipeline fails with the attached error messages.

The stdout_batch_job.txt file is the captured stdout of my LSF batch job.

I'm using caper run with the default conf file in ~/.caper/

Thanks for any insight!

[debug.tar.gz](https://github.com/ENCODE-DCC/atac-seq-pipeline/files/3368736/debug.tar.gz)
[stderr_not_empty.txt](https://github.com/ENCODE-DCC/atac-seq-pipeline/files/3368737/stderr_not_empty.txt)
[stdout_batch_job.txt](https://github.com/ENCODE-DCC/atac-seq-pipeline/files/3368738/stdout_batch_job.txt)

**Comment by leepc12** [comment]
After activating pipeline's conda environment,
Please check if tabix and bgzip are installed.
```
$ conda activate encode-atac-seq-pipeline
$ which bedtools
$ which tabix
$ which bgzip
```

**Comment by jwtobias** [comment]
Thanks for responding. All of my issues for that pipeline run are now resolved. I had to do three things. I expect number 1 fixed this particular problem, but there were other show-stoppers as well.

1. Keep irregular chromosome names.
2. Work around the problem that although things worked fine in interactive jobs, "conda activate" did not work in submitted jobs. I worked around this by initializing with "source activate" and by doing some linking in my conda binary directories.
3. Allocating an enormous amount of memory (400G) to the pipeline so jobs wouldn't fail when they hit the limit.

This last thing was likely due to the Xenopus Laevis genome build I was using (XenLae2 from UCSC) which had ~150K small chromosomes if I recall correctly.
